src/main/java/use_case/registration/RegistrationDataAccessInterface.java

```diff
@@ -0,0 +1,8 @@
+package use_case.registration;
+
+import entity.User;
+
```

Comment:
/**
     * Checks if the given username exists.
     * @param username the username to look for
     * @return true if a user with the given username exists; false otherwise
     */